spec/rails_helper.rb

```diff
@@ -18,6 +18,8 @@
 
 ActiveRecord::Migration.maintain_test_schema!
 
+Dir[File.expand_path("../support/**/*.rb", __FILE__)].each { |f| require f }
```

Comment:
You don't need this: put the helper in `spec/support/helpers` instead and it will autoload.